Add back `REVOKE ROLE role FROM user` in syntax table and other syntax updates (#747)
neo4j/neo4j-documentation: cypher/cypher-docs/src/docs/dev/ql/administration/security/user-management-syntax.asciidoc

  [[SET PASSWORD] CHANGE [NOT] REQUIRED]
  [SET STATUS {ACTIVE \| SUSPENDED}]
----

> | Create a new user, replacing an existing user if one exists with the same name.

Reviewer: same as above

Author: ```suggestion
| Create a new user, or if a user with the same name exists, replace it.
```
